systers_portal/templates/meetup/add_meetup.html

```diff
@@ -39,9 +39,9 @@
   })
 
   $('.timepicker').pickatime({
-    format: 'HH:i',
-    formatSubmit: 'HH:i',
+    format: 'H:i',
+    formatLabel: 'h:ia',
```

Comment:
Because when I display time `{{ meetup.time }}` it appears in 12 hours formate with day time period.